packages/property-grid/package.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "@bentley/property-grid-react",
+  "version": "1.0.0",
+  "description": "This package offers a React widget which provides access to the property grid within an iModel. Designed for use in an iModeljs 9-zone application",
+  "license": "MIT",
+  "main": "lib/property-grid-react.js",
+  "typings": "lib/property-grid-react",
+  "scripts": {
+    "copy:assets": "cpx \"./src/**/*.scss\" ./lib && cpx \"./public/**/*\" ./lib/public",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
+    "build": " tsc 1>&2 && npm run copy:assets && npm run pseudolocalize",
+    "clean": "rimraf lib",
+    "lint": "tslint -p . 1>&2",
```

Comment:
This currently doesn't do anything.  Did you mean to add a `tslint.json` or config to the `package.json`?

Reply:
Going to resolve this for now.  We need to revisit listing in all packages in the repo.

Reply:
added a tslint.json. Currently it just extends the tslint configuration found in `@bentley/build-tools/tslint-docs.json` and disables one rule. LMK if you have any recommendations for further modification here